### PR TITLE
English: services overview

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translate the services overview

### CODE REVIEW

File: source/localizable/diensten.html.erb [hunk 1]
```diff
@@ -6,67 +6,15 @@ title:
 description:
   nl: Defacto Software levert e-learning, implementatietrajecten, veilige hosting en security en koppelingen met vele andere (HR) systemen.
   de: Defacto Software bietet E-Learning, Implementierungen, safe Hosting und Security und Verbindungen zu vielen anderen (Quell-)Systemen.
+  en: Defacto Software delivers e-learning, implementation trajectories, safe hosting and security and integration with many other (HR) systems.
```
Comment (sn3p) [inline source/localizable/diensten.html.erb]: > E-Learning

File: locales/en.yml [hunk 2]
```diff
@@ -135,13 +139,18 @@ en:
   services:
     services:
       elearning:
+        desc: "With Defacto as an experienced and technically skilled partner, E-learning will be a success in your organisation."
         name: E-Learning
       hosting:
+        desc: "Having 25 years of experience with IT implementation at large (governmental) institutions, we know very well  that high quality hosting and high level information security are of an utmost importance."
         name: "Hosting & Security"
       implementaties:
+        desc: "Defacto has developed its own implementation method through which our project manager and consultants will safeguard the strategy and execution together with your management team."
```
Comment (sn3p) [inline locales/en.yml]: Suggestion:
> Defacto has developed its own methods through which we ensure a successful strategy and implementation while working close together with our clients.

File: locales/en.yml [hunk 3]
```diff
@@ -135,13 +139,18 @@ en:
...
         name: Implementations
       koppelingen:
+        desc: "CAPP LMS can smoothly be integrated  with other systems: information systems that include staff-related data, assessment databases, suppliers of e-learning and many more."
```
Comment (sn3p) [inline locales/en.yml]: Suggestion (not sure about the HR part):
> CAPP LMS can be integrated smoothly with other systems. For example information systems that include HR-related data, assessment databases, E-learning suppliers and more.

**Comment by RonaldDijkstra** [comment]
@mildaskl some responses to the copy for you to react on! 👍

File: locales/en.yml [hunk 4]
```diff
@@ -135,13 +139,18 @@ en:
   services:
     services:
       elearning:
+        desc: "With Defacto as an experienced and technically skilled partner, E-learning will be a success in your organisation."
         name: E-Learning
       hosting:
+        desc: "Having 25 years of experience with IT implementation at large (governmental) institutions, we know very well  that high quality hosting and high level information security are of an utmost importance."
```
Comment (mildaskl) [inline locales/en.yml]: The very well part has to be dropped for sure. Then the large (governmental) institutions has to go. Should be like governmental and enterprise institutions.

File: locales/en.yml (hunk 2 above)
Comment (mildaskl) [inline locales/en.yml]: Completely agree with the suggestion. The previous sentence (I believe) is a literal translation.

File: locales/en.yml (hunk 3 above)
Comment (mildaskl) [inline locales/en.yml]: Agreed.

File: source/localizable/diensten.html.erb (hunk 1 above)
Comment (mildaskl) [inline source/localizable/diensten.html.erb]: lets do E-Learning everywhere. Will have to upgrade the nl too then

**Comment by RonaldDijkstra** [comment]
Thank you @mildaskl

**Pull request merged**